apisix/plugins/slslog/rfc5424.lua

```diff
@@ -0,0 +1,104 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local LOG_EMERG     = 0       --  system is unusable
+local LOG_ALERT     = 1       --  action must be taken immediately
+local LOG_CRIT      = 2       --  critical conditions
+local LOG_ERR       = 3       --  error conditions
+local LOG_WARNING   = 4       --  warning conditions
+local LOG_NOTICE    = 5       --  normal but significant condition
+local LOG_INFO      = 6       --  informational
+local LOG_DEBUG     = 7       --  debug-level messages
+
+local LOG_KERN      = 0       --  kernel messages
+local LOG_USER      = 1       --  random user-level messages
+local LOG_MAIL      = 2       --  mail system
+local LOG_DAEMON    = 3       --  system daemons
+local LOG_AUTH      = 4       --  security/authorization messages
+local LOG_SYSLOG    = 5       --  messages generated internally by syslogd
+local LOG_LPR       = 6       --  line printer subsystem
+local LOG_NEWS      = 7       --  network news subsystem
+local LOG_UUCP      = 8       --  UUCP subsystem
+local LOG_CRON      = 9       --  clock daemon
+local LOG_AUTHPRIV  = 10      --  security/authorization messages (private)
+local LOG_FTP       = 11      --  FTP daemon
+local LOG_LOCAL0    = 16      --  reserved for local use
+local LOG_LOCAL1    = 17      --  reserved for local use
+local LOG_LOCAL2    = 18      --  reserved for local use
+local LOG_LOCAL3    = 19      --  reserved for local use
+local LOG_LOCAL4    = 20      --  reserved for local use
+local LOG_LOCAL5    = 21      --  reserved for local use
+local LOG_LOCAL6    = 22      --  reserved for local use
+local LOG_LOCAL7    = 23      --  reserved for local use
+
+local Facility = {
+    ["KERN"] = LOG_KERN,
```

Comment:
code style:

change to `KERN = LOG_KERN`, which is clearer

Reply:
fixed.